src/ECER.Managers.Registry/RegistrantHandlers.cs

```diff
@@ -86,6 +86,11 @@ public async Task<string> Handle(RegisterNewUserCommand request, CancellationTok
     }
     else
     {
+      registrants = await registrantRepository.Query(new RegistrantQuery
+      {
+        ByRegistrationNumber = request.Profile.RegistrationNumber,
+      }, cancellationToken);
+
```

Comment:
I think there's a potential flaw here. 

https://eccbc.atlassian.net/browse/ECER-1166 <-- account creation logic specifically item 2. 

if the user provides the registration number we should be matching by 
LastName
DOB
and Registration number 

I think lines 89 -> 93 would only check for a matching registration number. 

ex. 
User1
Dob: 01Jan1991
LastName: Me
RegistrationNumber: 123 

newUser puts in
DOB: FakeDOB 
LastName: Fake
registrationNumber 123 <-- we just use this to match

so newUser gets linked to User1